packages/blocks/shuffle/.eslintrc.js

```diff
@@ -5,6 +5,17 @@ module.exports = {
     project: ["tsconfig.json"],
   },
   rules: {
+    "no-restricted-imports": [
+      "error",
+      {
+        paths: [
+          {
+            name: "@mui/material/Button",
+            importNames: [],
+          },
+        ],
+      },
+    ],
```

Comment:
Why is the import to this not an error then? I'm confused what's trying to be achieved here.

Reply:
I guess my point is we've decided we don't want to import `Button` because we have a wrapper, so we should have a comment where we do import it explaining why. But I don't feel that strongly so can leave that.